### DISCUSSION TITLE
202302 | DCE 5.0 攻坚语录集 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#813] [Announcements]
# 202302 | DCE 5.0 攻坚语录集 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/blogs/peter/

**Comment by yowenter** [comment]
飞翔的 Peter Pan !!!
